docs-src/api-overview.md

```diff
@@ -79,6 +96,128 @@ BackendPolicy. For more information on what may be configured with this resource
 in the future, refer to the related [GitHub
 issue](https://github.com/kubernetes-sigs/service-apis/issues/196).
 
+### Route binding
+
+When a Route binds to a Gateway it represents configuration that is applied on
+the Gateway that configures the underlying load balancer or proxy. How and which
+Routes bind to Gateways is controlled by the resources themselves. Route and
+Gateway resources have built-in controls to permit or constrain how they select
+each other. This is useful for enforcing organizational policies for how Routes
+are exposed and on which Gateways. Consider the following example:
+
+> A Kubernetes cluster admin has deployed a Gateway “shared-gw” in the “Infra”
+> Namespace to be used by different application teams for exposing their
+> applications outside the cluster. Teams A and B (in Namespaces “A” and “B”
+> respectively) bind their Routes to this Gateway. They are unaware of each other
+> and as long as their Route rules do not conflict with each other they can
+> continue operating in isolation. Team C has special networking needs (perhaps
+> performance, security, or criticality) and they need a dedicated Gateway to
+> proxy their application to the outside world. Team C deploys their own Gateway 
+> “dedicated-gw”  in the “C” Namespace that can only be used by apps in the "C" 
+> Namespace.
+
+![schema](images/gateway-route-binding.png)
+
+There is a lot of flexibility in how Routes can bind to Gateways to achieve
+different organizational policies and scopes of responsibility. These are 
+different relationships that Gateways and Routes can have:
+
+- **One-to-one** - A Gateway and Route may be deployed and used by a single 
+  owner and have a one-to-one relationship. Team C is an example of this. This 
+  is also how the Ingress resource works.
```

Comment:
This statement has the potential for confusion. Some of the Ingress controller take multiple (mostly all) Ingress resource of a specific class and configure a single proxy. I understand that cloud LBs don't do this, it would be best to remove this.

Reply:
yeah good point - I removed the comparison with Ingress